rules_gapic/python/py_gapic.bzl

```diff
@@ -0,0 +1,170 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//rules_gapic:gapic.bzl", "GapicInfo", "gapic_srcjar", "unzipped_srcjar")
+
+def _py_gapic_postprocessed_srcjar_impl(ctx):
+    gapic_srcjar = ctx.file.gapic_srcjar
+    formatter = ctx.executable.formatter
+    output_dir_name = ctx.label.name
+
+    output_main = ctx.outputs.main
+    output_test = ctx.outputs.test
+    output_pkg = ctx.outputs.pkg
+    outputs = [output_main, output_test, output_pkg]
+
+    output_dir_path = "%s/%s" % (output_main.dirname, output_dir_name)
+
+    # Note the script is more complicated than it intuitively should be because of limitations
+    # inherent to bazel execution environment: no absolute paths allowed, the generated artifacts
+    # must ensure uniqueness within a build.
+    script = """
+    unzip -q {gapic_srcjar} -d {output_dir_path}
+    {formatter} -q {output_dir_path}
+    pushd {output_dir_path}
+    zip -q -r {output_dir_name}-pkg.srcjar nox.py setup.py setup.cfg docs MANIFEST.in README.rst LICENSE
```

Comment:
minor: please consider using wildcards for things like this (something like `*.py` instead of `nox.py setup.py`). 

The reason for this is because usually it is easier to keep up with changes form generator (if they add a new file there is a lower chance that we will need to update the rule as well). The flipside of it is that we may mistakenly grab files, which should not be grabbed, but it is less likely (usually we want everything).

Reply:
With wildcards, it grabs the wrong files